cognite/powerops/preprocessor/cogreader.py

```diff
@@ -130,7 +104,7 @@ def get_extra_files_metadata(self) -> List[dict[str, str]]:
             return []
 
     @staticmethod
-    def file_metadata_to_dict(file_metadata):
+    def file_metadata_to_dict(file_metadata) -> dict[str, Union[str, int]]:
```

Comment:
if you do `from __future__ import annotations` at the top you are porting in 3.10 type hints so you can write.
```suggestion
    def file_metadata_to_dict(file_metadata) -> dict[str, str | int]:
```